doc/sphinxman/source/scf.rst

```diff
@@ -745,6 +751,33 @@ To avoid this, either set |scf__df_basis_scf| to an auxiliary
 basis set defined for all atoms in the system, or set |scf__df_scf_guess|
 to false, which disables this acceleration entirely.
 
+COSX Exchange
+~~~~~~~~~~~~~
+
+The semi-numerical COSX algorithm (described in [Neese:2009:98]_) is so-called
+because the two-electron ERIs are evaluated analytically over one electron 
+coordinate and numerically over the other electron coordinate. The numerical 
+integration is performed on standard DFT quadrature grids, which are described
+in :ref:`DFT`. Both the accuracy of the COSX algorithm and also the computational
+cost are directly determined by the size of the integration grid, so selection
+of the grid is important. This COSX implementation uses two separate grids.
+The SCF algorithm is first converged on a smaller grid, followed by a final SCF
+iteration on a larger grid. The size of the initial grid is controlled by the
+keywords |scf__cosx_radial_points_initial| and |scf__cosx_spherical_points_initial|. The final
+grid  is controlled by |scf__cosx_radial_points_final| and
+|scf__cosx_spherical_points_final|.
+Defaults have been chosen for both grids that balance cost and accuracy.
+Screening thresholds over integrals, densities, and basis extents are set
+with the |scf__cosx_ints_tolerance|, |scf__cosx_density_tolerance|, and
+|scf__cosx_basis_tolerance| keywords, respectively. The |scf__cosx_incfock|
```

Comment:
I'm not sure `cosx_basis_tolerance` is a descriptive keyword?

Reply:
That's fair. I added some more information about the screening keywords